coldfront/core/project/views.py

```diff
@@ -441,6 +446,10 @@ def post(self, request, *args, **kwargs):
         end_date = datetime.datetime.now()
         project.status = project_status_archive
         project.save()
+
+        # project signals
+        project_archive.send(sender=self.__class__,project_obj=project_obj)
```

Comment:
This fails with the following error: 
```
    project_archive.send(sender=self.__class__,project_obj=project_obj)
                                                           ^^^^^^^^^^^
NameError: name 'project_obj' is not defined.
```

This should be `project_archive.send(sender=self.__class__,project_obj=project)`